### PR TITLE
Fix  NPE in `learner_updated_at_path` when student has run yet. [#104…

### PR DESCRIPTION
**Pull request by knowuh**
…427626]

Added tests for the controller.  Could / should add tests for Coffeescript.

https://www.pivotaltracker.com/story/show/104427626

The main consumer of this controller endpoint is for student reports:
`app/assets/javascripts/student_lightbox_report.js.coffee.erb`

To QA this,  it might be necessary to create a new learner (either by joining a new class or something similar)

For Local QA I simply nulled-out the `last_run` field  for my test Report::Learner.

### CODE REVIEW

File: spec/controllers/report/learner_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe Report::LearnerController do
+
+  def newLearner(learner_stubs)
```
Comment (scytacki) [inline spec/controllers/report/learner_controller_spec.rb]: javascript camel case bleeding over to ruby :)

File: spec/controllers/report/learner_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      describe 'html format' do
+        it "should return a string containing formatted time" do
+          get :updated_at, :format => :html, :id => learner.id
+          response.status.should eq 200
+          response.body.should match '30758400'
+        end
+      end
+    end
+
+    describe "when the leraner hasn't run anything yet" do
```
Comment (scytacki) [inline spec/controllers/report/learner_controller_spec.rb]: leraner typo

**Comment by scytacki** [comment]
Noah, thanks for fixing this.

**Comment by knowuh** [comment]
I was just going to file a bug report, but then realized it was way more efficient to start trying to fix it while I was in there.  Typos fixed.

**Pull request merged**